app/controllers/seller_gift_cards_controller.rb

```diff
@@ -15,7 +16,8 @@ def show
               :created_at,
               :updated_at,
               :expiration,
-              :single_use
+              :single_use,
+              'la.created_at as last_updated'
```

Comment:
I know that these are pretty much always the same as we create a new row for each redemption, but do you think there's any value here in using `updated_at` just in case someone does manually update a record in the db?